staff/ferran-calvo/precourse/proyecto8/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <h1 class="title blueBackground white">PASAPALABRA</h1>
+        <h2 id="timer"></h2>
+        <link rel="stylesheet" type="text/css" href="./index.css">
+    </head>
+    <script  src="./index.js"></script>
+    <body>
+        <div id="content">
```

Comment:
Hey Ferran, just a tip in general for hmtl. Try using the tag`<section></section>` in order for it to be more semantic and easier for screen readers!